web/packages/test/scripts/configure-substrate.sh

```diff
@@ -85,13 +85,24 @@ config_xcm_version() {
     send_governance_transact_from_relaychain $ASSET_HUB_PARAID "$call"
 }
 
+config_penpal() {
+    # System::set_storage(CustomizableAssetFromSystemAssetHub,Ethereum)
+    local call=0x00040440770800eb78be69c327d8334d0927607220020109079edaa802
```

Comment:
It's why we print the key value in https://github.com/Snowfork/polkadot-sdk/pull/137#discussion_r1565475630